test/upgrade/prober/wathola/sender/services.go

```diff
@@ -34,7 +34,10 @@ var log = config.Log
 var senderConfig = &config.Instance.Sender
 
 type sender struct {
+	// counter is the number of events successfully sent
```

Comment:
I think the naming is confusing here. IMO 'counter' should be renamed successCounter or sendSucCounter, and totalReq should be renamed to  sendCounter or sendTotalCounter.

Reply:
Those are in the original code and I typically try to avoid changing variable names of existing code. But Chris did suggest to change to sth like:
```
Sent int
Request int
```
Does it sound better?

Reply:
SendCount / RequestCount ?

Reply:
Renamed.